src/pages/IndexPage.vue

```diff
@@ -1,16 +1,48 @@
 <template>
-  <q-page class="flex flex-center">
-    <h1>
-      <em class="fa-solid fa-person-digging"></em>
-      {{ $t('project.name') }}
-    </h1>
-  </q-page>
+  <div class="flex flex-center main">
+    <div class="fit row justify-center items-center">
+      <q-img
```

Comment:
According to have a fully customizable application, we want to have more class to describe all customisable part.
Here it can be `class="logo"` and on the `<div class="flex flex-center main">` it's better to say `homepage` instead of main.

Reply:
Sorry rename `homepage` class to `home-page`